learning_resources_search/serializers.py

```diff
@@ -258,12 +258,13 @@ class ContentFileSearchRequestSerializer(SearchRequestSerializer):
         help_text="if the parameter starts with '-' the sort is in descending order",
     )
     content_category = StringArrayField(required=False, child=serializers.CharField())
-    topic = StringArrayField(required=False, child=serializers.CharField())
     aggregations = StringArrayField(
         required=False,
         child=serializers.ChoiceField(choices=CONTENT_FILE_AGGREGATIONS),
     )
     resource_type = serializers.ReadOnlyField(default=[CONTENT_FILE_TYPE])
+    run_id = StringArrayField(required=False, child=serializers.CharField())
+    resource_id = StringArrayField(required=False, child=serializers.CharField())
```

Comment:
Would
```suggestion
    run_id = StringArrayField(required=False, child=serializers.IntegerField())
    resource_id = StringArrayField(required=False, child=serializers.IntegerField())
```
make sense here?